js/custom-search--inline.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file
+ * Bootstrap dropdown on search and inline search.
+ */
+
+(function ($, Drupal) {
+  Drupal.behaviors.cdSearchInline = {
+    attach: function (context, settings) {
+      $(document).ready(function(){
+
+        // Apply focus to input when dropdown is shown.
+        $('.cd-search--inline').on('shown.bs.dropdown', function () {
+          $(this).find('#edit-search-api-views-fulltext').focus();
+        });
+
+        $('.cd-search--inline').on('hidden.bs.dropdown', function () {
+          $(this).find('#edit-search-api-views-fulltext').blur();
+        });
+
+        // Add class on submit button when input has focus.
+        $('.cd-search--inline .cd-search--inline__input').on('focus', function(e) {
+          $(this).parents().find('.cd-search--inline__submit').addClass('js-has-focus');
+        });
+
+        $('.cd-search--inline .cd-search--inline__input').on('blur', function(e) {
+          $(this).parents().find('.cd-search--inline__submit').removeClass('js-has-focus');
+        });
+
+        $('.cd-search--inline .cd-search--inline__input').on('focus', function(e){
+          $(this).parents().addClass('js-has-focus');
+        });
+
+        $('.cd-search--inline .cd-search--inline__input').on('blur', function(e){
+          $(this).parents().removeClass('js-has-focus');
```

Comment:
I believe it should be `$(this).parent()` because [`parents()` with an empty selector returns a null set](https://api.jquery.com/parents/)

Reply:
I used .parents() for the default search because the markup is a little different - the input and the submit are wrapped in their own divs. 
I've kept it for the default search but added a selector
https://github.com/UN-OCHA/ocha_basic/pull/54/commits/bf78484458be5191c7c15ba821dcc615ef05203f#diff-885b7a779228e2686c28beba26830186
and changed it to .parent() for the inline-search because that's correct.
Thanks!